### PR TITLE
feat(api, front): add code convention to README

### LINKED ISSUE #866: Se mettre d'accord sur le format de sortie des API
Mettre à jour le README selon le choix retenu

### PR DESCRIPTION
**Pull request by mxmeunier**
Close #866

### CODE REVIEW

**Review by VSerain — APPROVED** [review]

File: packages/api/README.md [hunk 1]
```diff
@@ -58,6 +58,24 @@ Vous pouvez utiliser docker pour simplifier l'installation de MongoDB avec les c
...
+
+On préférera donc `/utilisateur` à `/user` et `{ etablissement: [] }` à `establishment: []`.
+
+### Code Source
+
+Le format camelCase doit être utilisé pour toute définition de variable dans le code.
+
+La totalité du code source doit être en anglais. La création de variables à partir de retour de l'API devra se traduire en anglais.
+
+Exemple : `js const establishment = association.etablissement // retour API en français mais variable en anglais `
```
Comment (alice-telescoop) [inline packages/api/README.md:77]:
```suggestion
Exemple : `const establishment = association.etablissement // retour API en français mais variable en anglais `
```

File: packages/front/README.md [hunk 2]
```diff
@@ -19,6 +19,24 @@ npm install
...
+
+On préférera donc `/utilisateur` à `/user` et `{ etablissement: [] }` à `establishment: []`.
+
+### Code Source
+
+Le format camelCase doit être utilisé pour toute définition de variable dans le code.
+
+La totalité du code source doit être en anglais. La création de variables à partir de retour de l'API devra se traduire en anglais.
+
+Exemple : `js const establishment = association.etablissement // retour API en français mais variable en anglais `
```
Comment (alice-telescoop) [inline packages/front/README.md:38]:
```suggestion
Exemple : `const establishment = association.etablissement // retour API en français mais variable en anglais `
```

File: packages/front/README.md [hunk 3]
```diff
@@ -19,6 +19,24 @@ npm install
 
 -   Modifier les templates selon la note de version
 
+## Conventions de Code
+
+### API
+
+Le format des attributs de l'API est le snake_case.
+
+L'API retourne des données identifiées en français. Les attributs JSON seront donc écris en français pour faciliter la compréhension des utilisateurs. Il en va de même pour le nom des ressources HTTP.
```
Comment (alice-telescoop) [inline packages/front/README.md:28]:
```suggestion
L'API retourne des données identifiées en français. Les attributs JSON seront donc écrits en français pour faciliter la compréhension des utilisateurs. Il en va de même pour le nom des ressources HTTP.
```

File: packages/api/README.md [hunk 4]
```diff
@@ -58,6 +58,24 @@ Vous pouvez utiliser docker pour simplifier l'installation de MongoDB avec les c
 4. Faites ensuite une requête HTTP POST `localhost:8080/auth/reset-password` avec comme corps `{ "password": [your new password], "token": [token from step 3] }` pour mettre à jour votre mot de passe.
 5. Enfin, pour vous connecter, faite une requête HTTP POST `localhost:8080/auth/login` avec comme corps de requête `{ "email": [your email]}, "password": [password defined in step 4] }`
 
+## Conventions de Code
+
+### API
+
+Le format des attributs de l'API est le snake_case.
+
+L'API retourne des données identifiées en français. Les attributs JSON seront donc écris en français pour faciliter la compréhension des utilisateurs. Il en va de même pour le nom des ressources HTTP.
```
Comment (alice-telescoop) [inline packages/api/README.md:67]:
```suggestion
L'API retourne des données identifiées en français. Les attributs JSON seront donc écrits en français pour faciliter la compréhension des utilisateurs. Il en va de même pour le nom des ressources HTTP.
```

File: packages/api/README.md (hunk 1 above)
Comment (VSerain) [reply packages/api/README.md:77]: Done

File: packages/front/README.md (hunk 2 above)
Comment (VSerain) [reply packages/front/README.md:38]: Done

File: packages/api/README.md (hunk 4 above)
Comment (VSerain) [reply packages/api/README.md:67]: Done

File: packages/front/README.md (hunk 3 above)
Comment (VSerain) [reply packages/front/README.md]: Done

**Pull request merged**